qgepplugin/interlis/NOTES_INTERLIS_QGEP.md

```diff
@@ -0,0 +1,19 @@
+# Interlis / QGEP oddities
+
+## QGEP
+
+- `qgep_od.file.fk_data_media` doesn't have a FK constraint to `qgep_od.data_media.obj_id`
+
+## SIA405_Abwasser
+
+- `Deckel.Fabrikat` looks like a typo, shouldn't it be `Deckel.Fabrikant` ? (comment is `Name der Herstellerfirma`)
```

Comment:
@olivierdalang German is the leading language - we have to correct the English translation - can you make an issue?

Reply:
I re-checked the translation: Deckel.Fabrikat is translated as cover.brand in deepl (brand as an alternative valid translation) https://www.deepl.com/translator#de/en/Fabrikat. So I think this is ok like this and no correction is needed. e.g. Nestlé can be a company that produces goods and be also a brand.